GVFS/GVFS/CommandLine/PrefetchVerb.cs

```diff
@@ -45,9 +45,16 @@ public class PrefetchVerb : GVFSVerb.ForExistingEnlistment
             "folders-list",
             Required = false,
             Default = "",
-            HelpText = "A file containing line-delimited list of folders to fetch. Wildcards are not supported.")]
+            HelpText = "A file containing line-delimited list of folders to fetch. Wildcards are not supported. Specify \"stdin\" for stdin")]
         public string FoldersListFile { get; set; }
 
+        [Option(

```

Comment:
As mentioned above, this would be better as `--stdin-files`. Unless you are _also_ needing a `--files-list <input-file>`, don't add this parameter. It is a natural parallel to `--folders-list <input-file>`, but unless there is a need let's keep it out.

Reply:
I'll keep the --files-list, to be consistent with --folders-list, but as per your suggestion, will add another pair, --stdin-files and --stdin-folders